<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerOpenIGTLinkIFModule</class>
 <widget class="qSlicerWidget" name="qSlicerOpenIGTLinkIFModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <spacer name="VerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="ConnectorListFrame">
     <property name="text">
      <string>Connectors</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="ConnectorListLayout">
      <item>
       <widget class="qMRMLIGTLConnectorTreeView" name="ConnectorListView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ConnectorButtonFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="AddConnectorButton">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveConnectorButton">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HorizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="qSlicerIGTLConnectorPropertyWidget" name="ConnectorPropertyWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="IOConfigurationFrame">
     <property name="text">
      <string>I/O Configuration</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="qMRMLIGTLIOTreeView" name="IOTreeView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>true</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qSlicerIGTLIONodeSelectorWidget" name="IGTIONodeSelectorWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLIGTLConnectorTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLIGTLConnectorTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerIGTLConnectorPropertyWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerIGTLConnectorPropertyWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLIGTLIOTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLIGTLIOTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerIGTLIONodeSelectorWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerIGTLIONodeSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
